AISKU/Tests/nightwatch/run_nightwatch.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+  '1. Navigate to tests' : function (browser) {
+    var reportName = `tests_output/qunit_output_${browser.options.desiredCapabilities.browserName}${browser.options.desiredCapabilities.version || ''}.html`;
+    var ieAppendage = browser.options.desiredCapabilities.version;
+    ieAppendage = ieAppendage ? `?version=${ieAppendage}` : '';
+    browser
+      .url(`http://localhost:8000/Tests/Selenium/Tests.html${ieAppendage}`)
```

Comment:
why is this needed?

Reply:
I add a meta tag to `ie` tests so that I can emulate other version of ie. This is so I can grab the version config option I set in `nightwatch.json` and pass it along to `serve_nightwatch.js`

https://docs.microsoft.com/en-us/windows/desktop/win7appqual/use-the-meta-tag-to-ensure-future-compatibility